src/DotNetLightning.Core/Chain/KeysInterface.fs

```diff
@@ -60,7 +60,7 @@ type IKeysRepository =
     /// Must add funding pub key signature for to the PSBT *And* return the signature
     abstract member SignWithFundingPrivKey: psbt: PSBT -> TransactionSignature * PSBT
     /// Must add signature to the PSBT *And* return the signature
-    abstract member GenerateKeyFromBasepointAndSign: psbt: PSBT -> pubkey: PubKey -> perCommitmentPoint: PerCommitmentPoint -> TransactionSignature * PSBT
+    abstract member SignHtlcTx: psbt: PSBT -> perCommitmentPoint: PerCommitmentPoint -> TransactionSignature * PSBT
```

Comment:
same here about the commit message, add "Refactoring: "

Reply:
Fixed.